src/rules/at-import-no-partial-extension/index.js

```diff
@@ -23,6 +23,13 @@ export default function (on, options) {
     })
     if (!validOptions) { return }
 
+    result.warn((
```

Comment:
Nice. I just did the [same thing](https://github.com/stylelint/stylelint/pull/1430) over at stylelint.